### PR TITLE
German User guide 10.17.0

### PR DESCRIPTION
**Pull request by MWindel**

### CODE REVIEW

File: content/benutzerhandbuch/enterprise-tenant-de-bundle/managing-tenants.md [hunk 1]
```diff
@@ -235,7 +237,7 @@ Plattformadministratoren können die Anfragerate jedes Untermandanten über die
 
 * HTTP-Queue begrenzen - Begrenzung des HTTP-Anfragenpuffers für den Mandanten
 * HTTP-Anfragen begrenzen - Begrenzung der HTTP-Anfragen für den Mandanten pro Sekunde
-* Stream-Queue begrenzen - Begrenzung des MQTT-Anfragenpuffers für den Mandanten
+* Stream-Puffer begrenzen - Begrenzung des MQTT-Anfragenpuffers für den Mandanten
```
Comment (MWindel) [inline content/benutzerhandbuch/enterprise-tenant-de-bundle/managing-tenants.md:240]:
```suggestion
* Stream-Queue begrenzen - Begrenzung des MQTT-Anfragenpuffers für den Mandanten
```

File: content/benutzerhandbuch/enterprise-tenant-de-bundle/ms-data-broker.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Der ursprüngliche Data Broker wird vorerst neben dem Microservice-basierten Dat
 Für die gemeinsam genutzten Public-Cloud-Instanzen der {{< product-c8y-iot >}}-Plattform ist der Messaging Service ab Version 10.13 standardmäßig aktiviert, und der Microservice-basierte Data Broker kann auf Wunsch für einzelne Mandanten aktiviert werden, die bereits Zugriff auf den ursprünglichen Data Broker haben.
 Für dedizierte und selbst gehostete Instanzen sind der Messaging Service und der Microservice-basierte Data Broker für Version 10.10 und höher verfügbar, müssen aber explizit aktiviert werden.
 
-Bitte wenden Sie sich an den [Produkt-Support](/welcome/contacting-support/), um Näheres über die Verwendung von Messaging Service und Microservice-basiertem Data Broker in Ihrer {{< product-c8y-iot >}}-Umgebung zu erfahren.
+Bitte [wenden Sie sich an den Produkt-Support](/welcome/contacting-support/), um Näheres über die Verwendung von Messaging Service und Microservice-basiertem Data Broker in Ihrer {{< product-c8y-iot >}}-Umgebung zu erfahren.
```
Comment (MWindel) [inline content/benutzerhandbuch/enterprise-tenant-de-bundle/ms-data-broker.md:18]:
```suggestion
Bitte wenden Sie sich an den [Produkt-Support](/welcome/contacting-support/), um Näheres über die Verwendung von Messaging Service und Microservice-basiertem Data Broker in Ihrer {{< product-c8y-iot >}}-Umgebung zu erfahren.
```

File: content/benutzerhandbuch/administration-de-bundle/configuring-single-sign-on.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ title: Konfigurieren von Single Sign-On
 layout: redirect
 ---
 
-{{< product-c8y-iot >}} bietet Single-Sign-On-Funktionalität (SSO), die es dem Anwender ermöglicht, sich mit einem einzigen 3rd-Party-Autorisierungsserver über ein OAuth2-Protokoll, z. B. Azure Active Directory (ADD), anzumelden. Aktuell wird die Vergabe von Autorisierungscodes nur mit Access Tokens im JWT-Format unterstützt.
+{{< product-c8y-iot >}} bietet Single-Sign-On-Funktionalität (SSO), die es dem Anwender ermöglicht, sich mit einem einzigen 3rd-Party-Autorisierungsserver über ein OAuth2-Protokoll, z. B. Azure Active Directory (AAD), anzumelden. Aktuell wird die Vergabe von Autorisierungscodes nur mit Access Tokens im JWT-Format unterstützt.
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/benutzerhandbuch/administration-de-bundle/configuring-single-sign-on.md:7]: Done

File: content/benutzerhandbuch/administration-de-bundle/configuring-single-sign-on.md [hunk 4]
```diff
@@ -113,35 +113,35 @@ In diesem Fall sieht der JWT-Claim folgendermaßen aus:
  }
  ```
 
-Durch den "in"-Operator besteht die Möglichkeit zu verifizieren, ob ein Wert in einer Liste vorhanden ist. Werte können außerdem in andere Objekte eingebettet sein. Ein Punkt (".") im Schlüssel indiziert, dass es sich um ein eingebettetes Objekt handelt.
+Über den "in"-Operator besteht die Möglichkeit zu verifizieren, ob ein Wert in einer Liste vorhanden ist. Werte können außerdem in andere Objekte eingebettet sein. Ein Punkt (".") im Schlüssel indiziert, dass es sich um ein eingebettetes Objekt handelt.
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/benutzerhandbuch/administration-de-bundle/configuring-single-sign-on.md:116]:
```suggestion
Durch den "in"-Operator besteht die Möglichkeit zu verifizieren, ob ein Wert in einer Liste vorhanden ist. Werte können außerdem in andere Objekte eingebettet sein. Ein Punkt (".") im Schlüssel indiziert, dass es sich um ein eingebettetes Objekt handelt.
```

File: content/benutzerhandbuch/administration-de-bundle/configuring-single-sign-on.md [hunk 5]
```diff
@@ -198,10 +198,10 @@ Platzhalter werden nicht auf Korrektheit geprüft. Jeder nicht erkannte oder fal
 
 Die Integration wurde erfolgreich mit Azure AD getestet. Die Konfigurationsschritte finden Sie unter [https://docs.microsoft.com/de-de/azure/active-directory/develop/v1-protocols-oauth-code](https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-protocols-oauth-code).
 
-Die folgenden Schritte verdeutlichen, wie Azure AD (Azure Active Directory) für SSO in {{< product-c8y-iot >}} verwendet werden kann.
+Die folgenden Schritte zeigen, wie Azure AD (Azure Active Directory) in {{< product-c8y-iot >}} für SSO verwendet werden kann.
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/benutzerhandbuch/administration-de-bundle/configuring-single-sign-on.md:201]:
```suggestion
Die folgenden Schritte verdeutlichen, wie Azure AD (Azure Active Directory)  für SSO in {{< product-c8y-iot >}} verwendet werden kann.
```

File: content/benutzerhandbuch/administration-de-bundle/managing-applications.md [hunk 6]
```diff
@@ -257,10 +246,10 @@ Auf der Registerkarte **Plugins** können Sie Plugins hinzufügen und entfernen.
 
 Klicken Sie einfach auf die Anwendung oder auf das Menüsymbol rechts neben einem Eintrag und anschließend auf **Bearbeiten**.
 
-In der Registerkarte **Attribute** können einige Felder bearbeitet werden, abhängig vom Typ der Anwendung (siehe [Anwendungsattribute](#application-properties)).
+In der Registerkarte **Attribute** können einige Felder bearbeitet werden, abhängig vom Typ der Anwendung.
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/benutzerhandbuch/administration-de-bundle/managing-applications.md:249]:
```suggestion
In der Registerkarte **Attribute** können einige Felder bearbeitet werden, abhängig vom Typ der Anwendung (siehe [Anwendungsattribute](#application-properties)).
```

File: content/benutzerhandbuch/device-management-de-bundle/device-details.md [hunk 7]
```diff
@@ -302,13 +325,13 @@ So ist etwa im Screenshot unten ein Diagramm für Temperaturmesswerte mit zwei v
...
+Wenn ein Diagramm Messwerte mit verschiedenen Series enthält, wird pro Series eine Y-Achse dargestellt. Im Beispiel oben werden die Temperaturdaten zweier Sensoren aufgezeichnet, nämlich "sensors-1" und "sensor-2" mit der gleichen Einheit wie °C. Hier werden Messwerte verschiedener Sensoren als 
 separate "Series"-Daten kategorisiert. Die Messwerte der jeweiligen Sensoren werden unter verschiedenen Series-Namen gespeichert (identisch mit den Sensornamen), und somit werden hier zwei Achsen dargestellt.
 Gehören die Messwerte derselben Series an, wird hingegen nur eine Y-Achse dargestellt.
 
 Bewegen Sie den Mauszeiger über den Graphen, um detaillierte Informationen zu den Messwerten anzuzeigen. Neben dem Mauszeiger wird ein Tooltip mit Details zum jeweiligen Messwert angezeigt (der Tooltip rastet bei dem am nächsten liegenden Messwert ein).
 
-{{< c8y-admon-info >}}
+> {{< c8y-admon-info >}}
```
Comment (MWindel) [inline content/benutzerhandbuch/device-management-de-bundle/device-details.md:334]:
```suggestion
{{< c8y-admon-info >}}
```

File: content/benutzerhandbuch/enterprise-tenant-de-bundle/timeseries.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
+---
+weight: 120
+title: Erweiterter Support für Zeitreihen
+layout: redirect
+aliases:
+  - /benutzerhandbuch/enterprise-tenant-de/#timeseries
+---
+
+Der {{< product-c8y-iot >}} Betriebsspeicher bietet erweiterten Support für Zeitreihen (sogenannte Zeitreihensammlungen) für Messdaten. Der folgende Abschnitt fasst zusammen, wie sich diese Funktion aktivieren/deaktivieren lässt.
```
Comment (MWindel) [inline content/benutzerhandbuch/enterprise-tenant-de-bundle/timeseries.md:9]:
```suggestion
Der {{< product-c8y-iot >}} Betriebsspeicher bietet erweiterten Support für Zeitreihen (sogenannte Zeitreihensammlungen) für Messdaten. Im Folgenden wird beschrieben, wie sich diese Funktion aktivieren/deaktivieren lässt.
```

File: content/benutzerhandbuch/administration-de-bundle/managing-applications.md [hunk 9]
```diff
@@ -172,18 +164,15 @@ Das Duplizieren einer Anwendung ist erforderlich, wenn Sie eine abonnierte Anwen
...
     {{< /c8y-admon-info >}}
 5. Klicken Sie abschließend **Duplizieren**, um die Anwendung zu erstellen.
 
 {{< c8y-admon-info >}}
-Falls die Anwendung für diesen Mandanten abonniert wurde, steht ein weiterer Umschalter, **Abonnierte Anwendungen überschreiben**, zur Verfügung. Wenn Sie diesen Umschalter aktivieren, werden die Werte für Name, Schlüssel und Pfad aus der Originalanwendung geerbt und Ihre duplizierte Anwendung überschreibt die abonnierte Anwendung. Deaktivieren Sie die Option, um die Werte zu ändern.<br><br><img src="/images/benutzerhandbuch/Administration/admin-application-duplicate-3.png" alt="Duplicate application">
+Falls die Anwendung für diesen Mandanten abonniert wurde, steht ein weiterer Umschalter, nämlich **Abonnierte Anwendungen überschreiben**, zur Verfügung. Wenn Sie diesen Umschalter aktivieren, werden die Werte für Name, Schlüssel und Pfad aus der Originalanwendung geerbt und Ihre duplizierte Anwendung überschreibt die abonnierte Anwendung. Deaktivieren Sie die Option, um die Werte zu ändern.<br><br><img src="/images/benutzerhandbuch/Administration/admin-application-duplicate-3.png" alt="Duplicate application">
```
Comment (MWindel) [inline content/benutzerhandbuch/administration-de-bundle/managing-applications.md:175]:
```suggestion
Falls die Anwendung für diesen Mandanten abonniert wurde, steht ein weiterer Umschalter, **Abonnierte Anwendungen überschreiben**, zur Verfügung. Wenn Sie diesen Umschalter aktivieren, werden die Werte für Name, Schlüssel und Pfad aus der Originalanwendung geerbt und Ihre duplizierte Anwendung überschreibt die abonnierte Anwendung. Deaktivieren Sie die Option, um die Werte zu ändern.<br><br><img src="/images/benutzerhandbuch/Administration/admin-application-duplicate-3.png" alt="Duplicate application">
```

File: content/benutzerhandbuch/device-management-de-bundle/device-details.md [hunk 10]
```diff
@@ -279,7 +302,7 @@ Die Loginformationen des Geräts werden abgefragt.
 
 ![Logs tab](/images/benutzerhandbuch/DeviceManagement/devmgmt-devices-logs.png)
 
-Das Abfragen der Logdaten von einem Gerät kann einige Zeit in Anspruch nehmen.
+>Das Abfragen der Logdaten von einem Gerät kann einige Zeit in Anspruch nehmen.
```
Comment (MWindel) [inline content/benutzerhandbuch/device-management-de-bundle/device-details.md:305]: Das Abfragen der Logdaten von einem Gerät kann einige Zeit in Anspruch nehmen.
```

Comment (MWindel) [inline content/benutzerhandbuch/device-management-de-bundle/device-details.md:305]:
```suggestion
Das Abfragen der Logdaten von einem Gerät kann einige Zeit in Anspruch nehmen.
```

File: content/benutzerhandbuch/administration-de-bundle/data-retention.md [hunk 11]
```diff
@@ -62,22 +62,26 @@ Weitere Informationen zu den Feldern finden Sie unter [So fügen Sie eine Datenh
...
-<img src="/images/benutzerhandbuch/Administration/admin-retention-rules-delete.png" alt="Delete retention rule">
+Bewegen Sie die Maus über die Zeile mit der Regel, die Sie löschen möchten, und klicken auf das rechts erscheinende Löschen-Symbol.
 
 {{< c8y-admon-info >}}
 Alle Datenhaltungsregeln werden sequenziell und unabhängig voneinander ausgeführt. Wenn es zwei Datenhaltungsregeln gibt, von denen eine spezifischere mit einem höheren maximalen Alter eine Untermenge von den Dokumenten definiert, die durch eine allgemeinere Regel mit einem niedrigeren maximalen Alter definiert werden, wird alles so abgearbeitet, als gäbe es nur eine einzige, allgemeinere Regel.
 
 Betrachtet man beispielsweise die beiden folgenden Regeln:
 
-<img src="/images/benutzerhandbuch/Administration/admin-retention-rules-commspec1.png" alt="Retention rules"/>
+| **Datentyp** | **Fragmenttyp** | **Typ**        | **Quelle** | **Maximales Alter** |
```
[the remark this answers is not in the document]
Comment (dianaolives) [reply content/benutzerhandbuch/administration-de-bundle/data-retention.md:72]: Layout seems okay when viewed in Hugo.

File: content/benutzerhandbuch/administration-de-bundle/data-retention.md [hunk 12]
```diff
@@ -62,22 +62,26 @@ Weitere Informationen zu den Feldern finden Sie unter [So fügen Sie eine Datenh
...
+|---------------|-------------------|-----------------|------------|-----------------|
+| MESSUNG   | *                 | c8y_Temperature | *          | 30 Tage         |
+| MESSUNG   | *                 | c8y_Temperature | 12345      | 60 Tage         |
 
 Alle Messwerte vom Typ `c8y_Temperature`, die älter als 30 Tage sind, werden entfernt, einschließlich der Messwerte, bei denen die Quelle `12345` entspricht.
 
 Wenn jedoch die folgenden Datenhaltungsregeln definiert wurden:
 
-<img src="/images/benutzerhandbuch/Administration/admin-retention-rules-commspec2.png" alt="Retention rules"/>
+| **Datentyp** | **Fragmenttyp** | **Typ**        | **Quelle** | **Maximales Alter** |
```
[the remark this answers is not in the document]
Comment (dianaolives) [reply content/benutzerhandbuch/administration-de-bundle/data-retention.md:81]: @MWindel Layout seems okay when viewed in Hugo.

File: content/benutzerhandbuch/device-management-de-bundle/simulator.md [hunk 13]
```diff
@@ -85,12 +83,10 @@ Die Messwertanweisungen beziehen sich auf ein Fragment. Fragmente werden verwend
 
 Die Anweisung "Warten" erfordert einen einzelnen Wert für die Dauer des Wartens in Sekunden.
 
-![Sleep](/images/benutzerhandbuch/DeviceManagement/devmgmt-simulator-sleep.png)
-
 #### So fügen Sie eine Anweisung hinzu
 
 1. Klicken Sie auf **Anweisung hinzufügen**, um dem Simulator eine neue Anweisung hinzuzufügen.
-2. Wählen Sie im darauf folgenden Dialog eine Nachricht aus der Auswahlliste.
+2.  Wählen Sie im darauf folgenden Dialog eine Nachricht aus der Auswahlliste.
```
Comment (MWindel) [inline content/benutzerhandbuch/device-management-de-bundle/simulator.md:89]:
```suggestion
2. Wählen Sie im darauf folgenden Dialog eine Nachricht aus der Auswahlliste.
```

File: content/benutzerhandbuch/cockpit-de-bundle/managing-assets.md [hunk 14]
```diff
@@ -164,14 +164,12 @@ Bevor Sie ein Gerät zu einer Asset-Hierarchie hinzufügen können, muss das Ger
 
 Führen Sie die folgenden Schritte aus, um einer Gruppe Geräte zuzuweisen:
 
-1. Wählen Sie im Navigator eine Gruppe aus dem Menü **Gruppe** und öffnen Sie dann die Registerkarte **Kind-Assets**.
+1. Wählen Sie im Navigator eine Gruppe aus dem Menü **Gruppe** und öffnen Sie dann die Seite **Kind-Assets**.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/cockpit-de-bundle/managing-assets.md:167]: As far as I can see this is not a page but a tab. I remember that you cleaned that up?

File: content/benutzerhandbuch/cockpit-de-bundle/managing-assets.md [hunk 15]
```diff
@@ -164,14 +164,12 @@ Bevor Sie ein Gerät zu einer Asset-Hierarchie hinzufügen können, muss das Ger
...
 2. Klicken Sie auf **Geräte zuweisen** rechts in der oberen Menüleiste.
 3. Wählen Sie in der Liste die hinzuzufügenden Geräte aus. Sie können Filter anwenden, um die Anzahl der angezeigten Geräte zu reduzieren.
 4. Klicken Sie auf **Zuweisen**, um die ausgewählten Geräte zuzuweisen.
 
-![Assign devices](/images/benutzerhandbuch/cockpit/cockpit-group-assign.png)
-
-Die Geräte werden der ausgewählten Gruppe zugewiesen und als Kind-Assets auf der Registerkarte **Kind-Assets** angezeigt.
+Die Geräte werden der ausgewählten Gruppe zugewiesen und als Kind-Assets auf der Seite **Kind-Assets** angezeigt.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/cockpit-de-bundle/managing-assets.md:172]: Same here, should be a tab. But please double-check in the English text and in the UI.

File: content/users-guide/cockpit-bundle/data-explorer.md [hunk 16]
```diff
@@ -230,8 +230,6 @@ To use this function, first a dashboard must be created. For details on dashboar
 
 To create a widget from the data explorer of in the navigator, click **More...** in the top menu bar and select **Send as a widget to report** from the context menu.
 
-![Data explorer add events](/images/users-guide/cockpit/cockpit-dataexplorer-sendwidget.png)
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/data-explorer.md:233]: Who decided to remove images from the English files here and below? Has this been approved by the dev team?

**Comment by BeateRixen** [comment]
@MWindel @dianaolives Looks good to me. Just some minor comments from my side.

File: content/benutzerhandbuch/administration-de-bundle/audit-logs.md [hunk 17]
```diff
@@ -54,7 +54,7 @@ Die Audit-Logs-Liste wird nicht automatisch aktualisiert, wenn eine Echtzeitaktu
 
 Um Logeinträge leichter durchsuchen zu können, können diese gefiltert werden nach:
 
- - Typ (Alarm, Operation, Smart Rule und mehr).
+ - Typ (Alarm, Operation, Smart Rule und mehr.).
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/benutzerhandbuch/administration-de-bundle/audit-logs.md:57]:
```suggestion
 - Typ (Alarm, Operation, Smart Rule und mehr).
```

File: content/benutzerhandbuch/administration-de-bundle/managing-applications.md [hunk 18]
```diff
@@ -303,11 +292,11 @@ Wurde eine gehostete Anwendung nicht korrekt gestartet, kann der Benutzer sie er
 1. Öffnen Sie die Anwendungsattribute für die entsprechende Anwendung, indem Sie darauf klicken.
 3. Öffnen Sie im Abschnitt **Aktivitätslog** das Kontextmenü der gewünschten Version über das Menüsymbol und wählen Sie **Archiv erneut aktivieren**.
 
-Die gewählte Anwendung wird erneut aktiviert, indem die entsprechenden Dateien aus dem Anwendungsverzeichnis entfernt werden und das gehostete Anwendungspaket erneut entpackt wird.
+Die gewählte Anwendung wird erneut aktiviert, indem die entsprechenden Dateien aus dem Anwendungsverzeichnis entfernt werden und das gehostete Anwendungspacket erneut entpackt wird.
```
[the remark this answers is not in the document]
Comment (MWindel) [reply content/benutzerhandbuch/administration-de-bundle/managing-applications.md:295]:
```suggestion
Die gewählte Anwendung wird erneut aktiviert, indem die entsprechenden Dateien aus dem Anwendungsverzeichnis entfernt werden und das gehostete Anwendungspaket erneut entpackt wird.
```

File: content/benutzerhandbuch/cockpit-de-bundle/managing-assets.md (hunk 14 above)
Comment (MWindel) [reply content/benutzerhandbuch/cockpit-de-bundle/managing-assets.md:167]:
```suggestion
1. Wählen Sie im Navigator eine Gruppe aus dem Menü **Gruppe** und öffnen Sie dann die Registerkarte **Kind-Assets**.
```

File: content/benutzerhandbuch/cockpit-de-bundle/managing-assets.md (hunk 15 above)
Comment (MWindel) [reply content/benutzerhandbuch/cockpit-de-bundle/managing-assets.md:172]:
```suggestion
Die Geräte werden der ausgewählten Gruppe zugewiesen und als Kind-Assets auf der Registerkarte **Kind-Assets** angezeigt.
```

File: content/users-guide/cockpit-bundle/data-explorer.md (hunk 16 above)
Comment (MWindel) [reply content/users-guide/cockpit-bundle/data-explorer.md:233]: These images where not in the 10.17.0 English documentation and when double checking in the 10.16. version, the screenshots only showed empty fields and menu entries, for which, if I remember correctly, decided not to take screenshots.

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**